plugins/importtext_procedure.cc

```diff
@@ -876,7 +882,7 @@ struct ImportTextProcedureWorkInstance
                            int chunkNum,
                            int64_t lineNum)
 	    {
-	        int64_t actualLineNum = lineNum + lineOffset;
+            int64_t actualLineNum = lineNum + lineOffset;
```

Comment:
I actually fixed the problem for my lines. There are a bunch of tabs in the file. I'll fix it post merge and have a commit that just replaces all the tabs so the whole file doesn't show up as modified in this pr